src/e2e-test/features/pipeline.canvas.actions.feature

```diff
@@ -0,0 +1,52 @@
+#
+# Copyright © 2022 Cask Data, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+#
+
+@Integration_Tests
+Feature: Pipeline Canvas - Actions
+
+  @PIPELINE_CANVAS_ACTIONS_TEST
+  Scenario: Correctly undo redo actions done by the user
+    When Open pipeline studio page
```

Comment:
You have `Given` tag for the function but here you have `When`

Reply:
nice catch!